tools/enso4igv/src/main/java/org/enso/tools/enso4igv/enso/EnsoYamlProject.java

```diff
@@ -291,7 +295,20 @@ protected Node copyNode(Node node) {
       try {
         var data = p.root.getFileObject("data", true);
         if (data != null) {
-          var dataNode = DataObject.find(data).getNodeDelegate().cloneNode();
+          var dataNode = new FilterNode(DataObject.find(data).getNodeDelegate()) {
+              @Override
+              public Transferable clipboardCopy() throws IOException {
```

Comment:
With 9013c354c66aefe2481dc24be0a61dc1527ceb14 one can copy/paste the data directory node to editor:

<img width="704" height="333" alt="obrazek" src="https://github.com/user-attachments/assets/c6c58b93-8f15-4bca-abe2-1c62602220b7" />

and it generates valid `Meta.Enso_Project.enso_project.data` code that can then be executed to obtain reference to the data directory.